src/components/ui/WorldMap.jsx

```
@@ -0,0 +1,133 @@
+import DottedMap from "dotted-map";
+import { motion } from "framer-motion";
+import { useEffect, useMemo, useRef, useState } from "react";
+
+export function WorldMap({ dots = [], lineColor = "#0ea5e9" }) {
```

ESLint

[error] 'dots' is missing in props validation

(react/prop-types)